php/Terminus/Models/Upstream.php

```diff
@@ -27,6 +27,14 @@ public function __construct($attributes, array $options = [])
         }
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function __toString()
+    {
+        return "{$this->id}: {$this->url}";
```

Comment:
`$this->url` returns the API URI for retrieving the data for this model. You want `$this->get('url')`, which accesses the attributes property's URL property.